### PR TITLE
[SW-1042] multichain account create modal

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/components/Modals/AccountCreate.vue [hunk 1]
```diff
@@ -7,86 +7,82 @@
...
+import { useI18n } from 'vue-i18n';
+import { MODAL_AE_ACCOUNT_CREATE } from '../../utils';
 import { useConnection, useModals } from '../../../composables';
 import BtnSubheader from '../buttons/BtnSubheader.vue';
 import Modal from '../Modal.vue';
-import Loader from '../Loader.vue';
-import PlusCircleIcon from '../../../icons/plus-circle-fill.svg?vue-component';
+import IconAE from '../../../icons/coin/aeternity.svg?vue-component';
+import IconBTC from '../../../icons/coin/bitcoin.svg?vue-component';
+import IconDOG from '../../../icons/coin/dogecoin.svg?vue-component';
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:49]:
```suggestion
import IconDOGE from '../../../icons/coin/dogecoin.svg?vue-component';
```

Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:49]: I'm not sure about the usage of uppercase letters in file names - couldn't it be standard CamelCase naming convention?

File: src/popup/components/Modals/AccountCreate.vue [hunk 2]
```diff
@@ -7,86 +7,82 @@
...
-      loading.value = true;
-      await store.dispatch('accounts/hdWallet/create');
-      loading.value = false;
+    async function createBTCAccount() {
       props.resolve();
     }
 
-    async function createMultisigAccount() {
-      await openModal(MODAL_MULTISIG_VAULT_CREATE);
+    async function createDOGAccount() {
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:73]: Same as above
```suggestion
    async function createDogeAccount() {
```

File: src/popup/components/Modals/AccountCreate.vue [hunk 3]
```diff
@@ -7,86 +7,82 @@
...
     return {
-      PlusCircleIcon,
       isOnline,
-      loading,
-      createPlainAccount,
-      createMultisigAccount,
+      createAEAccount,
+      createBTCAccount,
+      createDOGAccount,
+      t,
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:82]: what is `t`?

File: src/popup/components/Modals/AeternityAccountCreate.vue [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+<style lang="scss" scoped>
+@use '../../../styles/typography';
+
+.aeternity-account-create {
+  .message {
+    @extend %face-sans-16-medium;
+
+    line-height: 24px;
+    max-width: 280px;
```
Comment (pstachel) [inline src/popup/components/Modals/AeternityAccountCreate.vue:103]: Do you think the modal should be limited by the width and not adjust itself (with margins of course) to the window size?

File: src/popup/components/Modals/AccountCreate.vue [hunk 5]
```diff
@@ -3,90 +3,86 @@
...
+      <BtnSubheader
+        :header="$t('common.bitcoin')"
+        :subheader="$t('modals.createAccount.btnBTCSubtitle')"
+        :icon="iconBtc"
+        :disabled="!isOnline"
+        @click="createBtcAccount"
+      />
+      <BtnSubheader
+        :header="$t('common.dogecoin')"
+        :subheader="$t('modals.createAccount.btnDOGSubtitle')"
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:34]:
```suggestion
        :subheader="$t('modals.createAccount.btnDogeSubtitle')"
```

File: src/popup/components/Modals/AccountCreate.vue [hunk 6]
```diff
@@ -96,11 +92,15 @@ export default defineComponent({
 @use '../../../styles/typography';
 
 .account-create {
+  .content-wrapper {
+    padding-inline: 16px;
+    padding-bottom: 32px;
+  }
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:98]:
```suggestion
  .content-wrapper {
    padding: 0 16px 32px;
  }
```

File: src/popup/components/Modals/AccountCreate.vue [hunk 7]
```diff
@@ -96,11 +92,15 @@ export default defineComponent({
 @use '../../../styles/typography';
 
 .account-create {
+  .content-wrapper {
+    padding-inline: 16px;
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:96]: Isn't it our `--screen-padding-x`?

Comment (konradodo) [reply src/popup/components/Modals/AccountCreate.vue]: this variable doesn't work in this scope

File: src/popup/components/Modals/AccountCreate.vue [hunk 8]
```diff
@@ -3,91 +3,86 @@
...
+        :disabled="!isOnline"
+        @click="createBtcAccount"
+      />
+      <BtnSubheader
+        :header="$t('common.dogecoin')"
+        :subheader="$t('modals.createAccount.btnDogeSubtitle')"
+        :icon="iconDoge"
+        :disabled="!isOnline"
+        @click="createDogeAccount"
+      />
```
Comment (pstachel) [inline src/popup/components/Modals/AccountCreate.vue:38]: We decided not to include doge in this release - please remove/comment it out

**Review by pstachel — APPROVED** [review]

File: src/popup/locales/en.json [hunk 9]
```diff
@@ -112,12 +115,36 @@
...
+      "titleMultisig": "Add multisig vault",
+      "titleAeternityAccount": "Add æternity account",
+      "msg": "Please select blockchain to add an account address to",
+      "msgMultisig": "Generate new multisig account address and add it to your wallet.",
+      "msgAeternity": "Generate new account address or import accounts from harware wallet.",
+      "btnSubtitle": "Derive another account from your seed",
+      "btnAeSubtitle": "Add æternity account",
+      "btnBtcSubtitle": "Add Bitcoin account",
+      "btnDogeSubtitle": "Add Dogecoin account"
+    },
```
Comment (CedrikNikita) [inline src/popup/locales/en.json:147]: This is a duplicate.
```suggestion
```

File: src/popup/locales/en.json [hunk 10]
```diff
@@ -112,6 +115,18 @@
     "waitingToBeSent": "Waiting to be sent..."
   },
   "modals": {
+    "createAccount": {
+      "title": "Add account",
+      "titleMultisig": "Add multisig vault",
+      "titleAeternityAccount": "Add æternity account",
+      "msg": "Please select blockchain to add an account address to",
+      "msgMultisig": "Generate new multisig account address and add it to your wallet.",
+      "msgAeternity": "Generate new account address or import accounts from harware wallet.",
```
Comment (CedrikNikita) [inline src/popup/locales/en.json:124]: 1. It's a **hardware** wallet.
2. There are no support for hardware wallet right now. So the wording should be adjusted.

**Review by CedrikNikita — APPROVED** [review]

**Pull request merged**